ARExperiment/ARExperiment/Helpers/UIViewController+Helpers.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+extension UIViewController {
+    func setViewBackgroundColor(to color: UIColor) {
```

Comment:
is this that useful ?

Reply:
You are right, It had left over code for when it was doing more things than just changing the color.